src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/PreviewResult.cs

```diff
@@ -13,10 +16,12 @@ public class PreviewResult
 
         public IEnumerable<UpdatePreviewResult> Updated { get; }
 
-        public string Name { get; }
+        public IReadOnlyDictionary<string, SortedSet<string>>? NewSourceMappings { get; }
```

Comment:
Switched this out. My understanding is that it will enumerate the dictionary up front (in the ctor).

BTW, don't forget that XAML also binds to this property and that VS's "Find references" feature does not identify those.